.opencode/commands/write-acceptance-test.md

```
@@ -0,0 +1,8 @@
+---
+description: Write or complete a Cucumber acceptance test scenario for a given feature or endpoint.
+agent: build
+---
+
+Load the `write-acceptance-test` skill and follow its workflow to write or complete the acceptance test for: $ARGUMENTS
+
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)